Makefile

```diff
@@ -1,11 +1,13 @@
 CRYSTAL_PATH ?= $(shell pwd)/share/crystal-ic/src
 CRYSTAL_CONFIG_PATH ?= '$$ORIGIN/../share/crystal-ic/src'
+CRYSTAL_LIB_CONFIG_PATH ?= '$$ORIGIN/../lib/ic/share/crystal-ic/src'
 
 COMPILER ?= crystal
 FLAGS ?= --progress
-RELEASE_FLAGS ?= --progress --static
+RELEASE_FLAGS ?= --progress --static --release
```

Comment:
Wasn't sure if the lack of `--release` was intentional or not, adding here in case it was missed